examples/pytorch/text-classification/run_glue.py

```diff
@@ -389,7 +389,12 @@ def preprocess_function(examples):
             result["label"] = [(label_to_id[l] if l != -1 else -1) for l in examples["label"]]
         return result
 
-    datasets = datasets.map(preprocess_function, batched=True, load_from_cache_file=not data_args.overwrite_cache)
+    datasets = datasets.map(
+        preprocess_function,
+        batched=True,
+        load_from_cache_file=not data_args.overwrite_cache,
+        desc="Running tokenizer on dataset",
+    )
```

Comment:
Does it work for you? For me with latest `datasets` master it doesn't know of `desc`.

Here is to verify in isolation:

```
PYTHONPATH=src python -c 'from datasets import load_dataset; ds=load_dataset("glue", "cola"); ds.map(desc="")'
Reusing dataset glue (/home/stas/.cache/huggingface/datasets/glue/cola/1.0.0/dacbe3125aa31d7f70367a07a8a9e72a5a0bfeb5fc42e75c9db75b96da6053ad)
Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: map() got an unexpected keyword argument 'desc'
```

Reply:
Got the problem! Here, ds is not `Dataset` object but `DatasetDict` object. 
If you try `PYTHONPATH=src python -c 'from datasets import load_dataset; ds=load_dataset("glue", "cola"); ds["train"].map(desc="")'` that would work. I think I'll need to raise another PR for adding `desc` to DatasetDict objects as well [here](https://github.com/huggingface/datasets/blob/e08362256fb157c0b3038437fc0d7a0bbb50de5c/src/datasets/dataset_dict.py#L410)

Reply:
Awesome.

hope you manage to get the fix into 1.7.0 - otherwise we will have to wait until the next release.

Reply:
I've just raised the PR but 1.7.0 was released yesterday only :/

Reply:
that will be a problem. since there is no way to compare the git version.

Once datasets has merged your additional fix, let's come back here and set this PR to be merged once the required version is there. it will be easy to ensure because the CI will fail until the new version will be available.

it's also very possible that they will be making hotfixes, as is often the case with minor version bumps. So it might be sooner than we think.

Reply:
Sure, we can target a patch release too. Will let you know when that happens!